src/main/java/school/lemon/changerequest/java/introduction/hw2/Task3.java

```diff
@@ -3,6 +3,16 @@
 
 public class Task3 {
     public static String comboString(String s1, String s2) {
-        return "";
+        StringBuilder sb2 = new StringBuilder();
```

Comment:
Why do you use SB only for this case? By the way, It can be done without it there.